pypeit/core/skysub.py

```diff
@@ -618,7 +618,7 @@ def local_skysub_extract(sciimg, sciivar, tilts, waveimg, global_sky, rn2_img, t
     if std is True:
         chi2_sigrej = 100.0
         #sigrej_ceil = 1e10
-        sigrej = 25.0
+        sigrej = 50.0  # 25 wasn't enough for MagE 2x2 binning (probably undersampled)
```

Comment:
Maybe we should turn this off entirely?

Reply:
Sounds dangerous.  Am going to make that an Issue #717 but
leave it for now.